dockstore-client/src/main/java/io/dockstore/client/cli/nested/WesFile.java

```diff
@@ -51,6 +52,11 @@ public String getName() {
         // If the file was provisioned locally from an entry on Dockstore, we can remove the path to the temporary directory
         // from the file's absolute path to obtain a relative path.
         if (this.removablePrefix != null) {
+
+            if (!(Path.of(removablePrefix)).isAbsolute()) {
+                errorMessage("The provided removable prefix must be absolute", CLIENT_ERROR);
```

Comment:
Is a end-user going to understand this message? What is a removable prefix? I sort of get it from the comment on line 52, but an end-user won't see that.

Reply:
Fixed: https://github.com/dockstore/dockstore-cli/pull/124/commits/5b1389e7f400aa26dde0c1c70dc273386b02cdaa